flowzone.yml

```diff
@@ -1064,13 +1065,21 @@ jobs:
           - event_name: ${{ github.event_name }}
             event_action: ${{ github.event.action }}
 
-    permissions:
-      # Required to create comments on pull requests.
-      pull-requests: write
+    permissions: {}
 
     steps:
       - *rejectExternalPullRequest
       - *rejectInternalPullRequestTarget
+      - *rejectMissingSecrets
+      - <<: *getGitHubAppToken
+        with:
+          <<: *getGitHubAppTokenWith
+          permissions: >-
+            {
+              "metadata": "read",
+              "issues": "read",
+              "pull_requests": "write"
```

Comment:
It might be worth adding comments about why particular write permissions are needed